src/Tests/Tests.Benchmarking/BenchmarkProgram.cs

```diff
@@ -61,14 +62,15 @@ public static int Main(string[] arguments)
 
 		private static IConfig CreateDefaultConfig()
 		{
-			var jobs = new[]
+			var jobs = new List<Job>
 			{
-				Job.ShortRun.With(Runtime.Core).With(Jit.RyuJit),
-				Job.ShortRun.With(Runtime.Clr).With(Jit.RyuJit),
-				Job.ShortRun.With(Runtime.Clr).With(Jit.LegacyJit),
+				Job.MediumRun.With(CoreRuntime.Core30).With(Jit.RyuJit),
```

Comment:
Should we be testing all the TFMs that the client supports, or at least the latest patch of each minor?

Reply:
Fair comment, I'm okay with just running with the latest greatest on core.

For now at least I am more interested in the relative performance of branches vs mainline than the performance between platforms. Possibly something to consider in the long run I reckon.